src/main/java/com/google/sps/servlets/DishAnalysisServlet.java

```diff
@@ -73,8 +75,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       List<AnnotateImageResponse> responses = new_response.getResponsesList();
 
       // Initialize blocked catagories
-      Set<String> blockedCatagories = new HashSet<String>;
-      blockedCatagories = new HashSet<String>(Arrays.asList("Cuisine", "Dish", "Food", "Ingredient", "Salad", "Fried food"));
+      Set<String> blockedCatagories = new HashSet<String>(Arrays.asList("Cuisine", "Dish", "Food", "Ingredient", "Salad", "Fried food"));
```

Comment:
Why not use com.google.common.collect.ImmutableSet?

Reply:
Using ImmutableSet is a good idea, I added a commit which changes blockedCategories to use ImmutableSet.